src/librustc_typeck/check/mod.rs

```diff
@@ -3728,38 +3741,43 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
     }
 
     /// Given a function `Node`, return its `FnDecl` if it exists, or `None` otherwise.
-    fn get_node_fn_decl(&self, node: Node<'tcx>) -> Option<(&'tcx hir::FnDecl, ast::Ident, bool)> {
+    fn get_node_fn_decl(&self, node: Node<'tcx>)
+    -> Option<(&'tcx hir::FnDecl, ast::Ident, FnDeclType)> {
```

Comment:
I think the rustfmt indentation for this would be

```
    fn get_node_fn_decl(
        &self,
        node: Node<'tcx>,
    ) -> Option<(&'tcx hir::FnDecl, ast::Ident, FnDeclType)> {
```

but I'm not too bothered either way